<launch>

  <!-- get clock from vrep simulation -->
  <param name="use_sim_time" value="true"/>
  <node pkg="vrep_util" type="clock_publisher" name="vrep_clock" output="screen" />

  <node pkg="modelo_diferencial" type="pioneer_odometry" name="pioneer_odometry" output="screen" />
  
  <node pkg="lazo_abierto" type="trajectory_generator" name="trajectory_generator" output="screen" >
    <param name="stepping" type="double" value="0.1"/>
    <param name="total_time" type="double" value="20"/>
  </node>

  <node pkg="lazo_cerrado" type="trajectory_follower" name="trajectory_follower" output="screen" >
    <param name="goal_selection" type="str" value="PURSUIT_BASED"/>
    
    <param name="fixed_goal_x" type="double" value="3"/>
    <param name="fixed_goal_y" type="double" value="0"/>
    <param name="fixed_goal_a" type="double" value="-1.570796327"/> <!-- -3/2 * PI -->
  </node>

  <node pkg="lazo_cerrado" type="logger" name="logger" output="screen" />

</launch>
